lib/assert.js

```diff
@@ -294,7 +294,8 @@ assert.notStrictEqual = function notStrictEqual(actual, expected, message) {
 };
 
 function expectedException(actual, expected) {
-  if (!actual || !expected) {
+  // actual is guaranteed to be an Error object, but we need to check expected
```

Comment:
Nit: full stop at the end of the sentence. I assume actual doesn't need to be capitalised as it's a variable.